Makefile

```diff
@@ -240,19 +268,44 @@ generate-manifests: $(CONTROLLER_GEN) ## Generate manifests e.g. CRD, RBAC etc.
 		output:rbac:dir=$(RBAC_ROOT) \
 		rbac:roleName=manager-role
 
+.PHONY: generate-eks-bootstrap-manifests
+generate-eks-bootstrap-manifests: $(CONTROLLER_GEN)
+	$(CONTROLLER_GEN) \
+		paths=./bootstrap/eks/api/... \
+		paths=./bootstrap/eks/controllers/... \
+		crd:crdVersions=v1 \
+		rbac:roleName=manager-role \
+		output:crd:dir=./bootstrap/eks/config/crd/bases \
+		output:rbac:dir=./bootstrap/eks/config/rbac \
+		output:webhook:dir=./bootstrap/eks/config/webhook \
+		webhook
+
 ## --------------------------------------
 ## Docker
 ## --------------------------------------
 
 .PHONY: docker-build
-docker-build: ## Build the docker image for controller-manager
+docker-build:
```

Comment:
I don't have a default gcloud project so I had to set the `REGISTRY` to test this:
```
REGISTRY=gcr.io/foo make docker-build-core
```